Overall review of SDK docs 4.0.0 - closes #760 (#761)
LiskHQ/lisk-docs: modules/ROOT/pages/guides/app-development/launch.adoc

====
It is recommended to create a config object with all the options that differ from the default config options.
To check the default config options, go to the xref:{url_reference_config}[configuration page] or check it directly in the code.

> The default options for modules are as follows: +

Reviewer: ```suggestion
The default options for modules are located in: +
```

`framework/src/modules/MODULE_NAME/defaults/config.js`. +

> The default options for components are as follows: +

Reviewer: ```suggestion
The default options for components are located in: +
```
